main/options/options.cc

```diff
@@ -627,6 +627,8 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                                  "Packages which are allowed to ignore the restrictions set by `visible_to` "
                                  "and `export` directives",
                                  cxxopts::value<vector<string>>(), "<name>");
+    options.add_options(section)("packager-layer", "Valid layer names for packages.",
+                                 cxxopts::value<vector<string>>()->default_value("library,application"), "<layerName>");
```

Comment:
```suggestion
                                 cxxopts::value<vector<string>>()->default_value("library,application"), "<layer-name>");
```

This is the case that we tend to use in the `--help` output.